src/coreComponents/mesh/generators/PAMELAMeshGenerator.cpp

```diff
@@ -167,7 +168,19 @@ std::vector< int > getPamelaNodeOrder( PAMELA::ELEMENTS::TYPE const type )
     case PAMELA::ELEMENTS::TYPE::VTK_TETRA: return { 0, 1, 2, 3 };
     case PAMELA::ELEMENTS::TYPE::VTK_PYRAMID: return { 0, 1, 2, 3, 4 };
     case PAMELA::ELEMENTS::TYPE::VTK_WEDGE: return { 0, 3, 1, 4, 2, 5 };
-    case PAMELA::ELEMENTS::TYPE::VTK_HEXAHEDRON: return { 0, 1, 3, 2, 4, 5, 7, 6 };
+    case PAMELA::ELEMENTS::TYPE::VTK_HEXAHEDRON:
+    {
+      // if the reverseZ option is on, we have to switch the node ordering
+      // (top nodes become bottom nodes) to make sure that the hex volume is positive
+      if( isZReverse )
+      {
+        return { 4, 5, 7, 6, 0, 1, 3, 2 };
+      }
+      else
+      {
+        return { 0, 1, 3, 2, 4, 5, 7, 6 };
+      }
+    }
```

Comment:
```suggestion
return izReverse ? { 4, 5, 7, 6, 0, 1, 3, 2 } : { 0, 1, 3, 2, 4, 5, 7, 6 };
```
?

Reply:
Thanks for the suggestion, I like it, but the compiler says:
```
/g/g90/hamon1/geosx/GEOSX/src/coreComponents/mesh/generators/PAMELAMeshGenerator.cpp:175:26: error:                                                                                                             
      initializer list cannot be used on the right hand side of operator '?'                                                                                                                                    
      return izReverse ? { 4, 5, 7, 6, 0, 1, 3, 2 } : { 0, 1, 3, 2, 4, 5, 7, 6 };                                                                                                                               
                         ^~~~~~~~~~~~~~~~~~~~~~~~~~                                        
```